web/wwwroot/css/site.css

```diff
@@ -58,6 +58,29 @@ input.invalid {
   background-color: blue;
 }
 
+.setWidth{
+  width: 50%;
+}
+
+.ProfileStyling{
+  white-space: nowrap;
+}
+
+.ProfileStyling .button .h4 {
+  display: inline;
+}
+
+.ProfileStyling button {
+  float: right;
+  text-align: right;
+  margin-left: 80%;
+}
+
+.bg-primary p-2 rounded-1 {
+  position: fixed;
+  right: 70%;
+}
+

```

Comment:
what are these classes for? Just out of curiosity. I'm hesitant to do too much with CSS as we're still waiting on designs, and a lot of stuff can be done with Bootstrap.

Reply:
I think Blanca added these for the buttons on the edit page before I took over this branch. I will redo the buttons using bootstrap! :)